src/main/java/me/rigamortis/seppuku/impl/module/movement/PullDownModule.java

```diff
@@ -0,0 +1,51 @@
+package me.rigamortis.seppuku.impl.module.movement;
+
+import me.rigamortis.seppuku.api.event.EventStageable;
+import me.rigamortis.seppuku.api.event.player.EventPlayerUpdate;
+import me.rigamortis.seppuku.api.module.Module;
+import me.rigamortis.seppuku.api.value.NumberValue;
+import net.minecraft.client.Minecraft;
+import team.stiff.pomelo.impl.annotated.handler.annotation.Listener;
+
+/**
+ * @author cookiedragon234
+ * (cookiedragon234 owns me and all)
+ * also buy backdoored client kk thx
```

Comment:
We will not be promoting third party software inside the source code of Seppuku.

Reply:
Furthermore, childish comments based around ego and relevancy will not be tolerated.  Many of the maintainers, including myself, understand the immature nature of communities based around projects but we will not accept it and do not think it's funny at all.

Reply:
Done